File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/views/login/PasswordRecoveryView.java
package app.editors.manager.mvp.views.login;

import app.editors.manager.mvp.views.base.BaseView;
import moxy.viewstate.strategy.OneExecutionStateStrategy;
import moxy.viewstate.strategy.StateStrategyType;

@StateStrategyType(OneExecutionStateStrategy.class)
public interface PasswordRecoveryView extends BaseView {
    void onPasswordRecoverySuccess(String email);
    void onEmailError();
}

File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/views/login/PasswordRecoveryView.kt
package app.editors.manager.mvp.views.login

import app.editors.manager.mvp.views.base.BaseView
import moxy.viewstate.strategy.OneExecutionStateStrategy
import moxy.viewstate.strategy.StateStrategyType

@StateStrategyType(OneExecutionStateStrategy::class)
interface PasswordRecoveryView : BaseView {
    fun onPasswordRecoverySuccess(email: String)
    fun onEmailError()
}